### PR TITLE
RAD-375 Order report templates by title

### PR DESCRIPTION
**Pull request by achabill**
## Description

- Order report templates by dcTermsTitle
- Edit template titles in MrrtReportTemplateServiceComponentTestDataset.xml to more meaningful names; *CT Cardiac Bypass Graft* and *CT Chest Pulmonary Embolism*

## Related Issue

see https://issues.openmrs.org/browse/RAD-375

## Checklist:

- [x] My pull request only contains one single commit.
- [x] My pull request is based on the latest master branch
  `git pull --rebase upstream master`.
- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.
- [x] My code follows the code style of this project.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/radiology/RadiologyConstants.java [hunk 1]
```diff
@@ -20,12 +20,15 @@
     /**
      * {@code GlobalProperty} property for the DICOM UID component used to describe the org root.
      *
-     * <p>Refer to DICOM Standard DICOM PS3.5 Chapter 9 Unique Identifiers.</p>
+     * <p>
```
Comment (teleivo) [inline api/src/main/java/org/openmrs/module/radiology/RadiologyConstants.java]: @achabill why are these in your diff?
you should not have 21 files in your diff. only 3 MrrtReportTemplateServiceComponentTestDataset.xml, HibernateMrrtReportTemplateDAO.java, MrrtReportTemplateServiceComponentTest.java

File: api/src/test/java/org/openmrs/module/radiology/report/template/MrrtReportTemplateServiceComponentTest.java [hunk 2]
```diff
@@ -145,9 +142,9 @@ private void setUpTemporaryFolder() throws IOException {
     }
     
     /**
-    * @see MrrtReportTemplateService#getMrrtReportTemplate(Integer)
-    * @verifies get template with given id
-    */
+     * @see MrrtReportTemplateService#getMrrtReportTemplate(Integer)
```
Comment (teleivo) [inline api/src/test/java/org/openmrs/module/radiology/report/template/MrrtReportTemplateServiceComponentTest.java]: these javadocs modifications should also not be in the diff.

**Comment by teleivo** [comment]
@achabill the changes of the xml dataset, the hibernateDAO and the service test are now as wanted but you still have these formatting changes that you need to get rid of. formatting is done by the maven formatter plugin for java files. so if you run "mvn clean install" it will format them accordingly.

**Comment by achabill** [comment]
@teleivo I do a `mvn clean install` but there is no `git diff`. Means formatting is applied correctly.

**Comment by teleivo** [comment]
@achabill please make your diff only contain the files you worked onnas already said. Read up in git and how to edit your commit and its changes. It night be that you made these formatting changes in your IDE and that they are not xaught by the maven formatter Plugin.

**Pull request merged**